cloud-sql/postgres/activerecord/config/environments/development.rb

```diff
@@ -58,4 +58,14 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # Use a Unix socket when INSTANCE_UNIX_SOCKET (e.g. /cloudsql/project:region:instance) is defined
+  if ENV["INSTANCE_UNIX_SOCKET"]
+    self.paths['config/database'] = 'config/database_unix.yml'
```

Comment:
`self.` is redundant. (Rubocop will complain about this.)

Also, please use double quotes rather than single quotes. (Rubocop will also complain about this.)

Same for additional cases below in this and following files.

Reply:
Done.